agent/exec/container/controller.go

```diff
@@ -194,6 +195,12 @@ func (r *controller) Wait(pctx context.Context) error {
 				// If we get here, something has gone wrong but we want to exit
 				// and report anyways.
 				return ErrContainerDestroyed
+			case "health_status: unhealthy":
```

Comment:
if health status changes, we can get noticed from events. So there is no need to poll health status by using a ticker.